beluga_tools/package.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>beluga_tools</name>
+  <version>2.0.2</version>
+
+  <description>Set of miscellaneous tools to work with beluga.</description>
+
+  <maintainer email="serraramiro@ekumenlabs.com">Ramiro Serra</maintainer>
+
+  <license>Apache License 2.0</license>
+
+
+  <!-- The released debian for h5py package when on humble seems to have a bug where it's incompatible with numpy's 2.0.0 version.
+  We will install newest package version using pip in the development images for this specific distro.
+  See:
+  https://github.com/h5py/h5py/issues/2353
+  -->
+  <!-- <exec_depend condition="$ROS_DISTRO != 'humble'">python3-h5py</exec_depend> -->
+  <exec_depend condition="$ROS_DISTRO != 'humble'">python3-h5py</exec_depend>
+  <exec_depend condition="$ROS_VERSION == 2">python3-matplotlib</exec_depend>
+  <exec_depend condition="$ROS_VERSION == 2">python3-numpy</exec_depend>
+  <exec_depend condition="$ROS_VERSION == 2">python3-scipy</exec_depend>
+
+  <!-- Distros newer than humble come with pip version > 23.0.1 and that doesn't allow mixing
+   python packages installed with pip outside a venv and installed using the
+   distro package manager (i.e. apt). Rosdep doesn't handle this in any way.
+   Therefore, users with newer pip version should install these manually.
+   Using beluga's development containers will already have these installed globally.
+   -->
+  <exec_depend condition="$ROS_DISTRO == 'humble'">python-plyfile-pip</exec_depend>
```

Comment:
@serraramiro1 fyi: if you `export PIP_BREAK_SYSTEM_PACKAGES=1` before `rosdep`, it won't break. `rosdep` is just a thin wrapper around `pip`.

Reply:
Done. Had to use `-E` since it seems we need several env variables for this to work. LMK if that's OK.